app/config/errors.php

```diff
@@ -75,17 +78,19 @@
     ],
     Exception::GENERAL_CURSOR_NOT_FOUND => [
         'name' => Exception::GENERAL_CURSOR_NOT_FOUND,
-        'description' => 'The cursor is invalid. This can happen if the item represented by the cursor has been deleted.',
+        'description' => 'The cursor used in pagination is invalid. This can happen if the item represented by the cursor has been deleted. Retry this request using another item from the previous as the cursor.',
```

Comment:
```suggestion
        'description' => 'The cursor used in pagination is invalid. This can happen if the item represented by the cursor has been deleted or doesn't exist. Retry this request using another item from the previous as the cursor.',
```
Also the last part `Retry this request using another item from the previous as the cursor.` is not very clear and looks like a mouthful to read & parse 👀